scripts/get-search-meta.ts

```diff
@@ -88,11 +88,19 @@ async function getSearchMeta() {
 
   for (const file of files) {
     let result: any[] = []
-    try {
-      result = await getMDXMeta(file)
-      json.push(...result)
-    } catch (error) {
-      console.log(error)
+    /**
+     * * Ignore "tutorial" file paths with an empty push
+     * * Remove this check when tutorials are live!
+     */
+    if (/[\\/]pages[\\/]tutorial/gi.test(file)) {
```

Comment:
Could we simplify this to make it easier to ignore certain pages in the future. I wonder if we can do something like `const excluded = ['pages/tutorial']` and then omit based on items in that array?

Another idea is to update https://github.com/chakra-ui/chakra-ui-docs/blob/main/contentlayer.config.ts#L116 and add an `excludeInSearch: true` to the `computedFields`, and then filter the files based on that.

Reply:
@segunadebayo I am not familiar with `contentlayer` but I have been working to understand it as I go! Based on what you suggested, it does not appear that adding `excludeInSearch` would work as expected, either because it is not an option in the config (type error) or it would not reach a component or script that would prevent the slugs from being added to `search-meta.json` (via frontmatter).

Since you suggested adding an array to check against, maybe this:
```js
/**
 * File paths to not be included in the search meta.
 *
 * This can be overall page sections (i.e. "/docs", "/tutorial", etc.) or specific files (i.e. "/guides/first-steps").
 */
const excludedSlugs = ['/tutorial']

for (const file of files) {
  let result: any[] = []

  // Windows OS: ensure file paths have forward slashes.
  const fileToPosixPath = posixPath(file)

  const isExcluded = excludedSlugs.find((excludedSlug) =>
    fileToPosixPath.includes(excludedSlug),
  )

  try {
    result = isExcluded ? null : await getMDXMeta(file)
    json.push(...result)
  } catch (error) {
    console.log(error)
  }
}
```